compiler/rustc_session/Cargo.toml

```diff
@@ -17,3 +17,6 @@ rustc_span = { path = "../rustc_span" }
 rustc_fs_util = { path = "../rustc_fs_util" }
 rustc_ast = { path = "../rustc_ast" }
 rustc_lint_defs = { path = "../rustc_lint_defs" }
+
+[target.'cfg(unix)'.dependencies]
+libc = "0.2"
```

Comment:
Does rustc_interface still need this dependency?

Reply:
No, it doesn't need this and `winapi`. Removed both.